src/plugins/vis_type_table_new/public/components/table_vis_grid_columns.tsx

```diff
@@ -18,26 +17,26 @@ export const getDataGridColumns = (
   event: IInterpreterRenderHandlers['event'],
   columnsWidth: ColumnWidth[]
 ) => {
-  const filterBucket = (rowIndex: number, columnIndex: number, negate: boolean) => {
-    const foramttedColumnId = cols[columnIndex].id;
-    const rawColumnIndex = table.columns.findIndex((col) => col.id === foramttedColumnId);
-    event({
-      name: 'filterBucket',
-      data: {
-        data: [
-          {
-            table: {
-              columns: table.columns,
-              rows,
-            },
-            row: rowIndex,
-            column: rawColumnIndex,
-          },
-        ],
-        negate,
-      },
-    });
-  };
+  // const filterBucket = (rowIndex: number, columnIndex: number, negate: boolean) => {
+  //   const foramttedColumnId = cols[columnIndex].id;
+  //   const rawColumnIndex = table.columns.findIndex((col) => col.id === foramttedColumnId);
+  //   event({
+  //     name: 'filterBucket',
+  //     data: {
+  //       data: [
+  //         {
+  //           table: {
+  //             columns: table.columns,
+  //             rows,
+  //           },
+  //           row: rowIndex,
+  //           column: rawColumnIndex,
+  //         },
+  //       ],
+  //       negate,
+  //     },
+  //   });
+  // };
```

Comment:
Can we skip this too? @ananzh is in the process of moving this to replace the existing table Vis and will likely be updating this function.

Reply:
Yes, no longer relevant.